plugins/bcm57xx/bcm57xx.quirk

```diff
@@ -0,0 +1,7 @@
+# Broadcom BCM5719
+[DeviceInstanceId=PCI\VEN_14E4&DEV_1657]
+Plugin = bcm57xx
+
+# Dell PCI card
+[DeviceInstanceId=PCI\VEN_14E4&DEV_1657&SUBSYS_94E41904]
```

Comment:
Are you sure this subsystem vendor ID isn't from the host system?  Most of the devices in my system pick up the host's subssystem vendor ID.

Example:
```
00:1f.5 Serial bus controller [0c80]: Intel Corporation Comet Lake SPI (flash) Controller [8086:02a4]
        Subsystem: Dell Comet Lake SPI (flash) Controller [1028:0962]
        Flags: fast devsel
        Memory at fe010000 (32-bit, non-prefetchable) [size=4K]
```

Reply:
> Are you sure this subsystem vendor ID isn't from the host system

I'm pretty sure the subsystem ID is part of the PCI spec for the device; I've seen other cards just change the subsys IDs when they are rebranded for OEMs, but the VID/PID remains the same so the kernel driver still binds correctly.

Reply:
There are populated and match the values in NVM